org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/javaeditor/codemining/CalleeJavaMethodParameterVisitor.java

```diff
@@ -104,25 +104,34 @@ public boolean visit(MethodInvocation methodInvocation) {
 			IMethodBinding methodBinding= methodInvocation.resolveMethodBinding();
 			if (methodBinding != null) {
 				IMethod method = resolveMethodBinding(methodBinding);
-				collectParameterNamesCodeMinings(method, arguments, methodBinding.isVarargs());
+				collectParameterNamesCodeMinings(method, arguments, methodBinding);
 			}
 		}
 		return super.visit(methodInvocation);
 	}
 
-	protected void collectParameterNamesCodeMinings(IMethod method, List<?> arguments, boolean isVarargs) {
+	protected void collectParameterNamesCodeMinings(IMethod method, List<?> arguments, IMethodBinding mbinding) {
```

Comment:
This methods has one part that uses only the `IMethod`, the other only the `IMethodBinding`. This should be a method overload with two different parameterTypes and the dispatch can be done in a third method.